models/articles.models.js

```diff
@@ -42,3 +42,16 @@ exports.selectAllArticles = () => {
     return rows;
     });
 }
+
+exports.updateArticleByArticleId = (inc_votes, article_id) => {
+    return db.query(`UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *;`, [inc_votes, article_id])
+    .then(({rows})=>{
+        if (rows.length === 0) {
+            return Promise.reject({status: 404, msg: "Article not found."})
+        }
+        return rows
```

Comment:
Ideally we should send back the zero index position here rather than in the controller for convention